packages/desktop-electron/index.js

```diff
@@ -175,6 +190,14 @@ async function createWindow() {
   clientWin = win;
 }
 
+function processUrl(url) {
+  if (!url.includes('localhost:') && !url.includes('app://')) {
+    shell.openExternal(url);
+    return true;
+  }
+  return false;
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** separation on concerns: it would be better to split this in two things - URL validation (returns bool) and the `shell.openExternal` operation. It will also then improve the readability of the code here.